apps/simd_op_check/Makefile

```diff
@@ -1,15 +1,17 @@
 CXX-host ?= c++
-CXX-arm-64-android ?= $(ANDROID_NDK_HOME)/toolchains/aarch64-linux-android-4.9/prebuilt/linux-x86_64/bin/aarch64-linux-android-c++
```

Comment:
Did you intend to commit the changes to the simd_op_check makefile on this branch?